client/my-sites/people/invite-people/index.jsx

```diff
@@ -245,6 +239,60 @@ class InvitePeople extends React.Component {
 		return tokens;
 	};
 
+	async sendInvites( siteId, usernamesOrEmails, role, message, isExternal ) {
+		try {
+			const response = await wpcom
+				.undocumented()
+				.sendInvites( siteId, usernamesOrEmails, role, message, isExternal );
+
+			const countValidationErrors = Object.keys( response.errors ).length;
+
+			if ( countValidationErrors ) {
+				let errorMessage;
+
+				if ( countValidationErrors === usernamesOrEmails.length ) {
+					errorMessage = this.props.translate(
+						'Invitation failed to send',
+						'Invitations failed to send',
+						{
+							count: countValidationErrors,
+							context: 'Displayed in a notice when all invitations failed to send.',
+						}
+					);
+				} else {
+					errorMessage = this.props.translate(
+						'An invitation failed to send',
+						'Some invitations failed to send',
+						{
+							count: countValidationErrors,
+							context: 'Displayed in a notice when some invitations failed to send.',
+						}
+					);
+				}
+
+				this.props.errorNotice( errorMessage );
+				this.props.recordTracksEventAction( 'calypso_invite_send_failed' );
+			} else {
+				this.props.successNotice(
+					this.props.translate( 'Invitation sent successfully', 'Invitations sent successfully', {
+						count: usernamesOrEmails.length,
+					} )
+				);
+				this.props.recordTracksEventAction( 'calypso_invite_send_success', { role } );
+			}
+
+			this.refreshFormState( response.errors, response.sent );
+		} catch ( error ) {
+			this.props.errorNotice(
+				this.props.translate(
+					"Sorry, we couldn't process your invitations. Please try again later."
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Sorry, we couldn't process your payment. Please try again later.' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/reduxify/invites-sent-new)